frontend/src/pages/agreements/list/AgreementTableRow.jsx

```diff
@@ -101,15 +99,13 @@ export const AgreementTableRow = ({ agreement }) => {
                         onClick={() => handleDeleteAgreement(agreement.id)}
                     />
 
-                    {(agreement.status === "DRAFT" || agreement.status === "UNDER_REVIEW") && (
```

Comment:
do we really want to remove this requirement?

Reply:
yeah, I haven't sort out how this is supposed to work in the new design yet, but I thought for Cliff's ticket it would just be removed.  I'd like something at the Agreement level for this, if we are removing status, then I don't know what it is.  Can we include something like "is_editible" or "is_locked" in the GET Agreement(s) response?